lambda_handlers/unit_tests/hubapi_list_test.py

```diff
@@ -0,0 +1,55 @@
+import os
```

Comment:
folder structure can be `tests/unit`. file name should start with `test`. hence you can rename it is `test_hubapi_list.py`